lib/plan/plan_itinerary_tabs.dart

```diff
@@ -275,8 +294,8 @@ class PlanItineraryTabPagesState extends State<PlanItineraryTabPages>
   Widget _buildExpandButton(BuildContext context) {
     return IconButton(
       icon: _isExpanded
-          ? Icon(Icons.keyboard_arrow_down, color: Colors.grey)
-          : Icon(Icons.keyboard_arrow_up, color: Colors.grey),
```

Comment:
The buttons in the top area are white. The collapse toggle button matches this style now.